### PR TITLE
Added more details and references

### PR DESCRIPTION
**Pull request by xsankar**
# PR Checklist 🚨

It is intended that only defined "*vulnerability entry leads*" should be able to create PR's (See [CODEOWNERS](https://github.com/OWASP/www-project-top-10-for-large-language-model-applications/blob/main/CODEOWNERS) and [Core Team](https://github.com/OWASP/www-project-top-10-for-large-language-model-applications/wiki/Core-Team))

If you are not a CODEOWNER for the entry of artifact, please see our Issues and Discussions boards

## Proposed changes

_Brief description_

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have x-referenced the appropriate issue this addresses within the [projects board](https://github.com/OWASP/www-project-top-10-for-large-language-model-applications/issues)
- [ ] I have applied the relevant [labels](https://github.com/OWASP/www-project-top-10-for-large-language-model-applications/labels) this PR addresses

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: 2_0_vulns/emerging_candidates/RetrievalAugmentedGeneration.md [hunk 1]
```diff
@@ -95,3 +101,5 @@ Information Classification: Tag and classify data within the knowledge base to c
 9. [Sentence Embedding Leaks More Information than You Expect: Generative Embedding Inversion Attack to Recover the Whole Sentence](https://arxiv.org/pdf/2305.03010)
 10. [Universal and Transferable Adversarial Attacks on Aligned Language Models](https://llm-attacks.org/)
 11. https://www.maginative.com/article/rlhf-in-the-spotlight-problems-and-limitations-with-a-key-ai-alignment-technique/)
+12. https://docs.aws.amazon.com/prescriptive-guidance/latest/llm-prompt-engineering-best-practices/best-practices.html#guardrails
```
Comment (GangGreenTemperTatum) [inline 2_0_vulns/emerging_candidates/RetrievalAugmentedGeneration.md]:
```suggestion
[AWS: Use guardrails](https://docs.aws.amazon.com/prescriptive-guidance/latest/llm-prompt-engineering-best-practices/best-practices.html#guardrails)
```

File: 2_0_vulns/emerging_candidates/RetrievalAugmentedGeneration.md [hunk 2]
```diff
@@ -95,3 +101,5 @@ Information Classification: Tag and classify data within the knowledge base to c
 9. [Sentence Embedding Leaks More Information than You Expect: Generative Embedding Inversion Attack to Recover the Whole Sentence](https://arxiv.org/pdf/2305.03010)
 10. [Universal and Transferable Adversarial Attacks on Aligned Language Models](https://llm-attacks.org/)
 11. https://www.maginative.com/article/rlhf-in-the-spotlight-problems-and-limitations-with-a-key-ai-alignment-technique/)
```
Comment (GangGreenTemperTatum) [inline 2_0_vulns/emerging_candidates/RetrievalAugmentedGeneration.md]:
```suggestion
[RLHF In the Spotlight: Problems and Limitations with Key AI Alignment Technique](https://www.maginative.com/article/rlhf-in-the-spotlight-problems-and-limitations-with-a-key-ai-alignment-technique/)
```

**Review by GangGreenTemperTatum — CHANGES_REQUESTED** [review]

**Comment by xsankar** [comment]
Minor edits on the reference list

**Comment by xsankar** [comment]
Minor edits on the reference list

File: 2_0_vulns/emerging_candidates/RetrievalAugmentedGeneration.md [hunk 3]
```diff
@@ -94,4 +100,6 @@ Information Classification: Tag and classify data within the knowledge base to c
 8. [Information Leakage in Embedding Models](https://arxiv.org/abs/2004.00053)
 9. [Sentence Embedding Leaks More Information than You Expect: Generative Embedding Inversion Attack to Recover the Whole Sentence](https://arxiv.org/pdf/2305.03010)
```
Comment (GangGreenTemperTatum) [inline 2_0_vulns/emerging_candidates/RetrievalAugmentedGeneration.md:101]:
```suggestion
9. [Sentence Embedding Leaks More Information than You Expect: Generative Embedding Inversion Attack to Recover the Whole Sentence](https://arxiv.org/pdf/2305.03010)
10. [Universal and Transferable Adversarial Attacks on Aligned Language Models](https://llm-attacks.org/)
11. [RLHF In the Spotlight: Problems and Limitations with Key AI Alignment Technique](https://www.maginative.com/article/rlhf-in-the-spotlight-problems-and-limitations-with-a-key-ai-alignment-technique/)
12. [AWS: Use guardrails](https://docs.aws.amazon.com/prescriptive-guidance/latest/llm-prompt-engineering-best-practices/best-practices.html#guardrails)
13. [Unleashing Worms and Extracting Data: Escalating the Outcome of Attacks against RAG-based Inference in Scale and Severity Using Jailbreaking](https://arxiv.org/abs/2409.08045)
```

**Review by GangGreenTemperTatum — CHANGES_REQUESTED** [review]
@xsankar , follow this step with my suggestion then good to go:

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

**Comment by xsankar** [comment]
Minor edits on the Reference List

**Review by GangGreenTemperTatum — APPROVED** [review]

**Pull request merged**